_saas-integrations/google-search-console/v1/google-search-console-v1.md

```diff
@@ -0,0 +1,187 @@
+      4. Click **CREATE**.
+
+  - title: "Enable {{ integration.display_name }} API"
+    anchor: "enable-apis"
+    content: |
+      1. Login to your **Google Cloud Platform**. Make sure you are inside the project that you will use for your {{ integration.display_name }} integration.
+      2. Select **APIs & Services** from the left-hand menu panel. This will take you to the **APIs & Services** dashboard.
+      3. In the dashboard, click **ENABLE APIS AND SERVICES**. You will be redirected to the **API Library**.
+      4. Search for {{ integration.display_name }} and select `{{ integration.display_name }} API`.
+      5. Click **ENABLE**.
+
+  - title: "Verify API domain"
+    anchor: "verify-api-domain"
+    content: |    
+      1. Staying in the **APIs & Services** section, click on **Domain verification** in the left-hand menu panel.
+      2. Click **Add domain**.
+      3. Add the domain you're using for this project.
+
+  - title: "API OAuth consent"
+    anchor: "oauth-consent"
+    content: |
+      1. Staying in the **APIs & Services** section, click on the **OAuth consent screen** in the left-hand menu panel.
+      2. Select your project.
+      3. Make the **Application type** `Public`.
+      4. Make the **Application name** `tap-google-search-console`.
+      5. Optionally, you can add an image for a logo to help you identify your app.
+      6. In the **Support email** field, enter your email that you will use for this app.
+      7. In the **Scope** section, click **Add scope**.
+      8. Add the following scope: `Google Search Console API ../auth/webmasters.readonly`.
+      9. In the **Authorized domains** section, add the domains that you added to {{ integration.display_name}} in [step 1](#set-up-console).
+      10. Click **SAVE**.
+      11. You will be taken to a screen that says **OAuth grant limits**. The default limit is 100 grants per minute, per day. You can request to raise the grant limit for your project by clicking **Raise limit**. You can also change the time interval grant resets. The options for time intervals are: `1h`, `6h`, `1d,`, `7d`, and `30d`.
+
+  - title: "Create OAuth credentials"
+    anchor: "create-credentials"
+    content: |
+      1. Staying in the **APIs & Services** section, click on **Credentials** in the left-hand menu panel.
+      2. Click on the **Create credentials** dropdown menu.
+      3. Select **OAuth client ID**.
+      4. In the **Create OAuth client ID** page, select **Web application** for the Application type.
+      5. In the **Name** field, enter `tap-google-search-console`.
+      6. In the **Authorized redirect URIs** section, add the domains that you added in [step 1](#set-up-console).
+      7. Click **CREATE**.
+
+  - title: "add integration"
+    content: |
+      4. In the **Site Urls** field, enter the URLs you added to your sitemap file in [step 1](#set-up-console). Make sure your URLs begin with `https://` or `http://`.
```

Comment:
This is close to what I meant, but not entirely what I was getting at. *How* should the URLs be listed in this field? Are they separated by commas, pipes, something else? What does the tap's README say?

Also - whatever the answer, it'll need to be updated here as well as the API object.